src/lib.rs

```diff
@@ -39,9 +42,11 @@ pub use crate::select::complex::{Kind as SelectComplexKind, Model as SelectCompl
 pub use crate::select::mini::{Kind as SelectMiniKind, Model as SelectMini};
 pub use crate::select::searchable::{Kind as SelectSearchableKind, Model as SelectSearchable};
 pub use crate::select::vec_searchable::Model as VecSelect;
+pub use crate::simple_ip_input::{Ip as IpInput, Model as SimpleIpInput};
+pub use crate::simple_port_input::{Model as SimplePortInput, Port as PortInput};
+pub use crate::simple_text_input::Model as SimpleTextInput;
```

Comment:
Names would be better to change like:

simple_ip_input::Ip -> ip_range_input::IpRange
simple_ip_input::Model -> IpRangeInput
simple_port_input::Port -> port_range_input::PortRange
simple_port_input::Model -> PortRangeInput
simple_text_input::Model-> TextInput

Reply:
For IpRange can I use the pre existing one for HostNetworkGroup? A major difference between mine and the HostNetworkGroup's struct is that my fields are start: Option<String> and end: Option<String> instead of start: String and end: String. I was wondering if it's okay to send an empty string instead of None and handle the empty string on nice-web or is that bad code practice?

Reply:
Using pre-existing code is a beneficial practice as long as the code is well-written. However, I'm uncertain if this is the case here. The decision to use the `Option` type depends on how you intend to utilize it. Without `Option`, an empty `String` can indicate either the absence of a specified value or incorrect input. If you want to distinguish between these two cases and treat emptiness differently from wrong input, then you need to use `Option` to separate them. On the other hand, if you use `IpAddr` instead of `String`, you are forced to use `Option` to imply emptiness, but you don't need to be concerned about handling wrong input specifically. Therefore, the first aspect to consider is whether you strongly prefer using `String` or not, and how you plan to handle incorrect input. Nevertheless, even if using `IpAddr` is technically correct, you can still choose to use `String` because it is more convenient wherever it is employed. In essence, it is challenging to determine the universally best approach. To make your decision, you should weigh the following factors: 1) Whether the absence of a value is significant or not, 2) how to manage exceptions, and 3) how easy it is to implement. Ultimately, you should opt for whichever approach you believe is the most suitable.

Reply:
done